src/mongosh/data/partners.json

```diff
@@ -114,14 +98,10 @@
         "description": "Members that found us with Greenville",
         "contacts": [
             {
-                "$oid": "aaaa00000000000000000019",
-                "name": "Sophia Smith",
-                "phone": "546-925-3228"
+                "$oid": "aaaa00000000000000000019"
             },
             {
-                "$oid": "aaaa00000000000000000020",
-                "name": "Mia Smith",
```

Comment:
Is it not necessary to track names and contact info? Is this a part of something else?

Reply:
Looks like it's in the person/people data, for example:
https://github.com/agile-learning-institute/mentorHub-mongodb/blob/95a38918dd673567a55558cb408690e05cac0e20/src/mongosh/data/partners.json#L99-L106
https://github.com/agile-learning-institute/mentorHub-mongodb/blob/b5baa44b75b9d7f754c7fd5a0cf7e04990b53534/src/mongosh/data/people.json#L577-L601)

Reply:
That is correct - if you want names and phone numbers (like the API returns) you will use a lookup() function when quering the data to fetch that from the person collection.

You can find out more about the patterns in [this article on schema best practices](https://www.mongodb.com/developer/products/mongodb/mongodb-schema-design-best-practices/) 

PS The change to the Partner schema and data was approved previously and inadvertently regressed. This was just returning those files to their previous state.

Reply:
Got it, approved